README.md

```diff
@@ -3,11 +3,11 @@ Repository for deploying Operator.sol to various EVM-based blockchains
 
 ## Environment Variables
 
-You need to setup a .env file in this folder to support the indidual scripts.
+You need to setup a .env file in this folder to support the individual scripts.
 
-    WEB3_INFURA_PROJECT_ID=...
-    ETHERSCAN_TOKEN=...
-    POLYGONSCAN_TOKEN=...
+    export WEB3_INFURA_PROJECT_ID=...
```

Comment:
I don't think `.env` files work with `export` statements. See https://github.com/theskumar/python-dotenv

Reply:
To be honest, I just saw patricks `.env` example for brownie. https://github.com/PatrickAlphaC/brownie_simple_storage/blob/main/.env

I did not use any of the variables since my deployment was on arbitrum testnet and could not find the API key for the arbitrum explorer (or any account creation to obtain an API key). If yours worked without `export` then lets roll with that.